From 38980562b0f48f2bf2a3052d130577eb18f75edf Mon Sep 17 00:00:00 2001
From: Kevin Wolf <kwolf@redhat.com>
Date: Thu, 31 Jul 2014 16:03:34 -0500
Subject: [CHANGE 12/31] qemu-iotests: Disable tests that require sample images
To: rhvirt-patches@redhat.com,
    jen@redhat.com

RH-Author: Kevin Wolf <kwolf@redhat.com>
Message-id: <1406822631-6570-13-git-send-email-kwolf@redhat.com>
Patchwork-id: 60375
O-Subject: [RHEL-6.6 qemu-kvm PATCH v3 12/29] qemu-iotests: Disable tests that require sample images
Bugzilla: 1122410
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Max Reitz <mreitz@redhat.com>

We don't have the binary images in the RHEL 6 tree, so disable the tests
using them and don't complain if the directory doesn't exist.

Signed-off-by: Kevin Wolf <kwolf@redhat.com>
Signed-off-by: jen <jen@redhat.com>
---
 tests/qemu-iotests/common.config | 22 ++++++++++++----------
 tests/qemu-iotests/group         | 12 ++++++------
 2 files changed, 18 insertions(+), 16 deletions(-)

diff --git a/tests/qemu-iotests/common.config b/tests/qemu-iotests/common.config
index bd6790b..20346ad 100644
--- a/tests/qemu-iotests/common.config
+++ b/tests/qemu-iotests/common.config
@@ -125,16 +125,18 @@ fi
 
 export TEST_DIR
 
-if [ -z "$SAMPLE_IMG_DIR" ]; then
-        SAMPLE_IMG_DIR="$source_iotests/sample_images"
-fi
-
-if [ ! -d "$SAMPLE_IMG_DIR" ]; then
-    echo "common.config: Error: \$SAMPLE_IMG_DIR ($SAMPLE_IMG_DIR) is not a directory"
-    exit 1
-fi
-
-export SAMPLE_IMG_DIR
+# No binary sample images in RHEL 6
+#
+#if [ -z "$SAMPLE_IMG_DIR" ]; then
+#        SAMPLE_IMG_DIR="$source_iotests/sample_images"
+#fi
+#
+#if [ ! -d "$SAMPLE_IMG_DIR" ]; then
+#    echo "common.config: Error: \$SAMPLE_IMG_DIR ($SAMPLE_IMG_DIR) is not a directory"
+#    exit 1
+#fi
+#
+#export SAMPLE_IMG_DIR
 
 _readlink()
 {
diff --git a/tests/qemu-iotests/group b/tests/qemu-iotests/group
index 6e67f61..13c5ceb 100644
--- a/tests/qemu-iotests/group
+++ b/tests/qemu-iotests/group
@@ -65,26 +65,26 @@
 056 rw auto backing
 057 rw auto
 058 rw auto quick
-059 rw auto quick
+# 059 rw auto quick -- requires sample images
 060 rw auto quick
 061 rw auto quick
 062 rw auto quick
 063 rw auto quick
-064 rw auto quick
+# 064 rw auto quick -- requires sample images
 065 rw auto quick
 066 rw auto quick
 067 rw auto quick
 068 rw auto quick
 069 rw auto quick
-070 rw auto quick
+# 070 rw auto quick -- requires sample images
 071 rw auto quick
 072 rw auto quick
 073 rw auto quick
 074 rw auto quick
-075 rw auto quick
-076 auto
+# 075 rw auto quick -- requires sample images
+# 076 auto -- requires sample images
 077 rw auto quick
-078 rw auto quick
+# 078 rw auto quick -- requires sample images
 079 rw auto
 080 rw auto
 081 rw auto quick
-- 
1.9.3

